tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/StringSanitizer.cs

```diff
@@ -25,11 +25,23 @@ public static void ConfirmValidRegex(string regex)
         }
 
 
+        /// <summary>
+        /// General purpose string replacement. Simple abstraction of string.Replace().
+        /// </summary>
+        /// <param name="inputValue">The name of the header we're operating against.</param>
+        /// <param name="targetValue">The substitution or whole new header value, depending on "regex" setting.</param>
+        /// <param name="replacementValue">The substitution or new header value, depending on the "targetValue" setting.</param>
+        /// <returns>An updated value of the input string, with replacement operations completed if necessary.</returns>
+        public static string ReplaceValue(string inputValue, string targetValue, string replacementValue)
+        {
+            return inputValue.Replace(targetValue, replacementValue);
```

Comment:
Betting on additional requests making this a bit more complicated, hence the one line abstraction.